Assets/MixedRealityToolkit/Providers/Hands/HandRay/HandRay.cs

```diff
@@ -18,6 +21,9 @@ public Ray Ray
             }
         }
 
+        /// <summary>
+        /// Check whether hand palm is angled in a way that hand rays should be used.
+        /// </summary>
         public bool ShouldShowRay
```

Comment:
nit: I would just simplify to /// <inheritdoc/> if you aren't changing the comment

Reply:
Same with update and others, it'll make the files smaller too